src/main/java/edu/uc/inventorymanager/dto/ItemStatus.java

```diff
@@ -3,6 +3,6 @@
 import lombok.Data;
 
 public @Data class ItemStatus {
-    private int id;
-    private String name;
+    private int itemStatusId;
+    private String itemStatusName;
```

Comment:
Cleaning variable names in accordance to Baeldung - Clean Code

See: dto.Item comments.